phpBB/phpbb/di/container_builder.php

```diff
@@ -135,6 +135,11 @@ public function get_container()
 			$config_cache = new ConfigCache($container_filename, defined('DEBUG'));
 			if ($this->use_cache && $config_cache->isFresh())
 			{
+				if ($this->use_extensions)
+				{
+					require $this->get_autoload_filename();
```

Comment:
Parenthesis, I guess.. Or is it okay like that? I don't know the coding guidelines here.